content/applications/finance/fiscal_localizations/colombia/workflows.rst

```diff
@@ -0,0 +1,95 @@
+=========
+Workflows
+=========
+
+Main workflows
+==============
+
+.. image:: workflows/electronic-invoice-workflow.png
```

Comment:
An image alone is not sufficient documentation.
On future PR, we'll we'll want to write all of this out (maybe on #4425)

For the sake of this restructure though, we can let it go for now.